packages/replay-internal/test/integration/coreHandlers/handleGlobalEvent.test.ts

```diff
@@ -133,8 +134,8 @@ describe('Integration | coreHandlers | handleGlobalEvent', () => {
 
   it('tags errors and transactions with replay id for session samples', async () => {
     const { replay, integration } = await resetSdkMock({});
-    // @ts-expect-error protected but ok to use for testing
-    integration._initialize();
+    integration['_initialize'](getClient()!);
```

Comment:
omg this syntax 😅  Not wrong at all, just JS/TS 🙄

Reply:
yeah, it is definitely weird, but I prefer this over having to do `@ts-expect-error` 😅